src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -324,9 +324,15 @@ export default {
 				FLSS_ORIGIN_LEND_FILTER
 			);
 
+			const nonBinaryFilterEnabled = this.$route.query?.nonBinaryFilter;
```

Comment:
I'm pretty sure this can be removed as well. My changes altered the source of the gender enum options, and they are now coming from FLSS. So at least in dev, nonbinary is included, and this special casing isn't needed.

Reply:
Gotcha. Please add a `TODO` above this block mentioning that this will need to be removed with the experiment. If there's a ticket to clean up the code afterwards, please mention the ticket number.